http/codegen/server.go

```diff
@@ -521,10 +530,8 @@ func {{ .HandlerInit }}(
 		}
 	{{- end }}
 	{{- if .Method.SkipResponseBodyEncodeDecode }}
-		if _, err := io.Copy(w, o.Body); err != nil {
-			if err := encodeError(ctx, w, err); err != nil {
-				errhandler(ctx, w, err)
-			}
+		if _, err := io.Copy(w, io.MultiReader(buf, o.Body)); err != nil {
```

Comment:
Is it permissible to read from o.Body again if it returned an io.EOF when reading into buf?

Reply:
I believe this should behave correctly but haven't tested that myself...

Reply:
Switching to bufio avoids the need for a multireader, and thus the edge case I worried about.